app/models/page_content_register.rb

```diff
@@ -26,6 +31,34 @@ def new_blocks_attributes
 
   private
 
+  def srcset_blocks(blk_attrs: [])
+    blk_attrs
+      .select { |b| b[:identifier] =~ home_page_tile_identifer }
+      .map do |b|
+      {
+        identifier: b[:identifier].sub('image', 'srcset'),
+        content: srcset_content(original_url: b[:content]).join(', ')
+      }
+    end
+  end
+
+  def available_styles
+    Comfy::Cms::File.new.file.styles
+      .reject { |k,_| [:cms_thumb, :cms_medium].include? k }
+      .map do |style, object|
+      {
+        style: style,
+        width: object.geometry.split('x').first
+      }
+    end
+  end
```

Comment:
Is this file PageContentRegister the best location and the responsible for available_styles method? I would be honest that I don't know what the best place is, but it is worth we talk about it. And ... what would happen to the test ([here](https://github.com/moneyadviceservice/cms/pull/383/files#diff-9d0f2839f3bd8a7272090a62dc1bfc7dR46)) if you add more styles?